src/pydata_sphinx_theme/assets/scripts/bootstrap.js

```diff
@@ -0,0 +1,16 @@
+/* Sphinx injects the html output with jquery and other javascript files.
+ * To enable Popper.js (and other jQuery plugins) to hook into the same instance of jQuery,
+ * jQuery is defined as a Webpack external, thus this import uses the externally defined jquery dependency.
```

Comment:
will this then break once Sphinx stops bundling jquery?

Reply:
yes but I need this PR to be merged first to then upgrade to bootstrap 5 (which is not dependent on JQuery anymore)